developer/js/bundle.sh

```diff
@@ -2,6 +2,13 @@
 
 set -eu
 
+## START STANDARD BUILD SCRIPT INCLUDE
+# adjust relative paths as necessary
+THIS_SCRIPT="$(greadlink -f "${BASH_SOURCE[0]}" 2>/dev/null || readlink -f "${BASH_SOURCE[0]}")"
+. "$(dirname "$THIS_SCRIPT")/../../resources/build/build-utils.sh"
+. "$KEYMAN_ROOT/resources/build/jq.inc.sh"
```

Comment:
... `jq.inc.sh` is part of the standard include, but not `shellHelperFunctions.sh`?  (In reference to related edits in other 🎡 PRs)  That doesn't seem right.

Reply:
Good catch. I'll move it down. Nothing special about jq.inc.sh. But trying to be more consistent on this!

Reply:
I'll make this consistent in a future PR.